README.md

```diff
@@ -102,3 +102,4 @@ module.exports = schema;
 [Re:dash Dashboard]: https://sql.telemetry.mozilla.org/
 [Infernyx Homepage]: https://github.com/tspurway/infernyx
 [Joi Homepage]: https://github.com/hapijs/joi
```

Comment:
It's not visible in the diff, but line 70 still directs the user to the Joi Homepage for details on field modifiers.